pep-0637.rst

```diff
@@ -354,12 +354,34 @@ The successful implementation of this PEP will result in the following behavior:
    This allows notations such as ``[:, *args, :]``, which could be treated 
    as ``[(slice(None), *args, slice(None))]``.
 
+   The following notation equivalence should be honored::
+
+       obj[*()]        
```

Comment:
Did you discuss this on python-ideas? I actually think the reason for rejecting `obj[]` (that it looks like a typo) doesn't apply here. Ditto for `obj[**{}]`.

Reply:
I'm in favor of allowing that too - my point above was mainly that _something_ needed to be specified, and then python-ideas could argue for a better specification.

Reply:
I was not sure about it... maybe we should let it pass, and treat it like obj[()], same for obj[**{}] is valid and equivalent to obj[()]